scripts/setup.py

```diff
@@ -78,13 +78,62 @@ def download_and_install_package(url_to_retrieve, filename):
             downloaded.
         filename: string. The name of the tar file.
     """
+
     python_utils.url_retrieve(url_to_retrieve, filename=filename)
     tar = tarfile.open(name=filename)
     tar.extractall(path=common.OPPIA_TOOLS_DIR)
     tar.close()
     os.remove(filename)
 
 
+def download_and_install_node_on_windows(url_to_retrieve, filename):
+    """Downloads and installs node in Oppia tools directory for Windows.
+
+    Args:
+        url_to_retrieve: string. The url to retrieve node.
+        filename: string. The name of the tar file.
+    """
+    python_utils.url_retrieve(url_to_retrieve, filename=filename)
+    subprocess.check_call(
+        ['powershell.exe', '-c', 'expand-archive',
```

Comment:
Is Powershell a dependency for the Windows installation? If so, is it on Windows computers by default or do we need to list it as a prerequisite in our setup instructions?

Reply:
It's a default installation, so we should be okay here.

Reply:
Although it is a default installation, only PowerShell 5.1 and above has the `Expand-Archive` cmdlet. This requires the Windows system to be on Windows 10 and has a version higher than 1607 (which was released in 2016). Otherwise, the user may need to install the newer version manually.

Reply:
Understood. Sounds like we should document it then, in the wiki?

Reply:
Yes, I can do this once the PR is merged.